src/main/java/io/stargate/sgv2/jsonapi/exception/mappers/WebApplicationExceptionMapper.java

```diff
@@ -42,6 +43,12 @@ public RestResponse<CommandResult> webApplicationExceptionMapper(WebApplicationE
     if (e instanceof NotFoundException) {
       return RestResponse.status(RestResponse.Status.NOT_FOUND, commandResult);
     }
+    // Return 415 for invalid Content-Type
+    if (e instanceof NotSupportedException
```

Comment:
It's odd we have to do this in 2 different places -- do you know which one is necessary for the test to pass? Is it possible to test both handlers?

Reply:
This is because our current Exception structure design.
So this failure happens before our command processing, and it is captured by this WebApplicationExceptionMapper, but we still need want to convert the Throwable to CommandResult, so the same logic goes to ThrowableToErrorMapper. 
Here we just want the status 415 to be added.